articles/112_generated_interfaces_cpp.md

```diff
@@ -103,7 +103,19 @@ All other constants are declared as `static const` members in the struct and the
 
 ### Constructors
 
-The *default constructor* initializes all members with their default value.
+The *default constructor* initializes all members with their default value; in case the field doesn't have a default value, the C++ default is used (generally 0 or the empty string).
```

Comment:
Is there a reference for the "C++ default" for a value? If so perhaps we should link to it, if not maybe it could be added to the table above where there is a mapping between our idl primitive types and the matching C++ type.

Reply:
Yeah, you are right, this language is actually too vague.  I'm going to update it to talk about value initialization instead, since that is what we want to emulate.